src/common/resources/voucher.js

```diff
@@ -0,0 +1,29 @@
+import { get } from 'lodash';
+
+import {
+  VOUCHERS_API,
+  VOUCHER_LINES_API,
+} from '../constants';
+import { BASE_RESOURCE } from './base';
+
+export const VOUCHER = {
+  ...BASE_RESOURCE,
+  path: VOUCHERS_API,
+  records: 'vouchers',
+  params: {
+    query: 'invoiceId==!{invoiceId}',
+  },
+};
+
+export const VOUCHER_LINES = {
```

Comment:
I think will have the same issue we had with related invocies

Reply:
@AndreiShumski2 it would be nice if you can check that. Also maybe it makes sense sticking to the one way of doing the same things

Reply:
Fixed, I'll check it again after [MODINVOICE-90](https://issues.folio.org/browse/MODINVOICE-90) is done